src/node_buffer.cc

```diff
@@ -1442,6 +1442,83 @@ void CopyArrayBuffer(const FunctionCallbackInfo<Value>& args) {
   memcpy(dest, src, bytes_to_copy);
 }
 
+// TODO(ronag): simdutf for this.
+// Write as many valid ascii bytes as possible to dst and return
+// the number of bytes written.
+uint32_t write_ascii(uint8_t* dst, const uint8_t* src, uint32_t size) {
```

Comment:
@lemire I think this is the API we're looking for in simdutf.

Reply:
simdutf::convert_latin1_to_utf8 would be perfect, but it doesn't have a limiter on the output length

Reply:
How about now?